From 0bef9f9e4893b31ef475853d0af7993ecbe57f70 Mon Sep 17 00:00:00 2001
From: John Zhuge <jzhuge@cloudera.com>
Date: Mon, 15 May 2017 16:33:58 -0700
Subject: [PATCH 2436/2848] HADOOP-14417. Update default SSL cipher list for
 KMS. Contributed by John Zhuge.

(cherry picked from commit 2be559b8429a2d8b9d924bb16a67ad67d9bd7f40)

Change-Id: Ic7a4097b02535c1395ba53e8dddb206a40c1e515
---
 .../hadoop-kms/src/main/libexec/kms-config.sh      |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/hadoop-common-project/hadoop-kms/src/main/libexec/kms-config.sh b/hadoop-common-project/hadoop-kms/src/main/libexec/kms-config.sh
index 009f9c4..210b87a 100644
--- a/hadoop-common-project/hadoop-kms/src/main/libexec/kms-config.sh
+++ b/hadoop-common-project/hadoop-kms/src/main/libexec/kms-config.sh
@@ -192,11 +192,6 @@ if [ "${KMS_SSL_CIPHERS}" = "" ]; then
   KMS_SSL_CIPHERS+=",TLS_RSA_WITH_AES_128_CBC_SHA256"
   KMS_SSL_CIPHERS+=",TLS_RSA_WITH_AES_128_CBC_SHA"
   KMS_SSL_CIPHERS+=",TLS_RSA_WITH_3DES_EDE_CBC_SHA"
-  KMS_SSL_CIPHERS+=",TLS_DHE_RSA_WITH_AES_256_CBC_SHA256"
-  KMS_SSL_CIPHERS+=",TLS_DHE_RSA_WITH_AES_256_CBC_SHA"
-  KMS_SSL_CIPHERS+=",TLS_DHE_RSA_WITH_AES_128_CBC_SHA256"
-  KMS_SSL_CIPHERS+=",TLS_DHE_RSA_WITH_AES_128_CBC_SHA"
-  KMS_SSL_CIPHERS+=",TLS_DHE_RSA_WITH_3DES_EDE_CBC_SHA"
   print "Setting KMS_SSL_CIPHERS:           ${KMS_SSL_CIPHERS}"
 else
   print "Using   KMS_SSL_CIPHERS:           ${KMS_SSL_CIPHERS}"
-- 
1.7.9.5

